package.json

```diff
@@ -19,12 +20,12 @@
   },
   "homepage": "https://github.com/IgnacioBarocchi/HumbleScraper#readme",
   "dependencies": {
-    "cheerio": "^1.0.0-rc.3",
-    "request": "^2.88.2"
+    "node-fetch": "^2.6.0"
   },
   "devDependencies": {
     "@types/jest": "^26.0.0",
     "@types/node": "^14.0.13",
+    "@types/node-fetch": "^2.5.7",
```

Comment:
for every non-typescript library that we add, we need to also load the @types for TS to recognize all its type definitions.

Reply:
👍